### PR TITLE
CDAP-4783 Documentation of Explore enable/disable endpoints

### PR DESCRIPTION
**Pull request by awholegunch**
Passes [Quick Build 3](http://builds.cask.co/browse/CDAP-DOB149-3).

Fix for https://issues.cask.co/browse/CDAP-4783.

Pages of interest:

- [Query RESTful API, "Enable and Disable Querying"](http://builds.cask.co/artifact/CDAP-DOB149/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/reference-manual/http-restful-api/query.html#enabling-and-disabling-querying)
- [Data Exploration](http://builds.cask.co/artifact/CDAP-DOB149/shared/build-3/Docs-HTML/3.4.0-SNAPSHOT/en/developers-manual/data-exploration/index.html)

### CODE REVIEW

File: cdap-docs/reference-manual/source/http-restful-api/query.rst [hunk 1]
```diff
@@ -421,3 +442,87 @@ if results for the ``query-handle`` are attempted to be downloaded again.
...
+  POST <base-url>/namespaces/<namespace>/data/explore/datasets/<dataset-name>/enable
+  POST <base-url>/namespaces/<namespace>/data/explore/datasets/<dataset-name>/disable
+
+For streams::
+
+  POST <base-url>/namespaces/<namespace>/data/explore/streams/<stream-name>/tables/<table-name>/enable
+  POST <base-url>/namespaces/<namespace>/data/explore/streams/<stream-name>/tables/<table-name>/disable
+
+Each of these endpoints returns a query handle that can be used |---| in the case of
+``enable`` |---| to :ref:`submit queries <http-restful-api-query-submitting>` and explore
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/query.rst]: you don't use the handle to submit queries.  It is only used to track status of the query.

File: cdap-docs/reference-manual/source/http-restful-api/query.rst [hunk 2]
```diff
@@ -421,3 +442,87 @@ if results for the ``query-handle`` are attempted to be downloaded again.
...
+     - Description
+   * - ``200 OK``
+     - The query execution was successfully initiated, and the body will contain the query-handle
+       used to identify the query in subsequent requests
+   * - ``404 Not Found``
+     - The query is not well-formed or contains an error such as a nonexistent table name
+
+.. rubric:: Comments
+
+If the query execution was successful, the body will contain a query handle that can be used to
```
Comment (albertshau) [inline cdap-docs/reference-manual/source/http-restful-api/query.rst]: the body will always contain a query handle, even if execution fails.

**Comment by albertshau** [comment]
couple comments

File: cdap-docs/reference-manual/source/http-restful-api/query.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/query.rst]: Revised.

File: cdap-docs/reference-manual/source/http-restful-api/query.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/reference-manual/source/http-restful-api/query.rst]: Modified (please check).

**Comment by albertshau** [comment]
lgtm

**Pull request merged**